docs/content/graphql-api-authorization.md

```diff
@@ -41,7 +41,7 @@ schema {
 }
 
 type Query {
-  userByID(id: String): Employee
+  employeeByID(id: String): Employee
```

Comment:
I realized while testing the changes in this PR that I had never corrected this typo in the GraphQL tutorial. :upside_down_face: